src/pfe/portal/modules/Templates.js

```diff
@@ -23,18 +23,21 @@ const log = new Logger('Templates.js');
 const DEFAULT_REPOSITORY_LIST = [
   {
     url: 'https://raw.githubusercontent.com/kabanero-io/codewind-templates/master/devfiles/index.json',
-    description: 'Standard Codewind templates',
+    description: 'Extension to develop in containers with an IDE or language of your choice.',
     enabled: true,
     protected: true,
     projectStyles: ['Codewind'],
     name: 'Default templates',
   },
 ];
 
+const kabaneroDescription = 'Kabanero, an open source project, brings together open source technologies into a microservices-based framework.' +
+'Kabanero builds cloud native applications ready for deployment onto Kubernetes and Knative.'
+
```

Comment:
I'm OK with either description listed here, but one suggestion from the ID team is, "Curated collections with end-to-end support, including Appsody stacks and Tekton pipelines, for deployment on OpenShift." If you don't like this one better, however, you can leave the description you already have. :)